### ISSUE TITLE
Flytta alla +2 på `bus_data.count` till modulkom.h

### ISSUE THREAD

**Issue by Gosha** [Grupp-5/robot#6]
Bl.a. dessa:
https://github.com/Grupp-5/robot/blob/Manuell_styrning/beslutsenhet/beslutsenhet.c#L37
https://github.com/Grupp-5/robot/blob/Manuell_styrning/sensorenhet/sensorenhet.c#L105
https://github.com/Grupp-5/robot/blob/Manuell_styrning/kommunikationsenhet/communication.c#L33

**Comment by erima694** [comment]
Menar du inte till common.h?

**Comment by Gosha** [comment]
Nej.. Jag tycker att modulkom ska göra det i koden typ, så att siffrorna
inte behöver ändras i common.h
